DuckDuckGo/OmniBar.swift

```diff
@@ -285,6 +286,8 @@ extension OmniBar: Themable {
     public func decorate(with theme: Theme) {
         backgroundColor = theme.barBackgroundColor
         tintColor = theme.barTintColor
+        
+        configureTextField()
```

Comment:
This fixes a bug: OmniBar's placeholder did not update correctly when switching themes.